packages/utils/src/englishConjugation/conjugateEnglish.ts

```diff
@@ -52,7 +76,40 @@ function buildPast(parts: EnglishVerbComponents): EnglishVerbTense {
 }
 
 function buildPresent(parts: EnglishVerbComponents): EnglishVerbTense {
-  const verbTense = {} as EnglishVerbTense;
+  const verbTense = {
+    continuous: {
+      I: `was ${parts.pastParticiple}`,
+      you: `were ${parts.pastParticiple}`,
+      it: `was ${parts.pastParticiple}`,
+      we: `were ${parts.pastParticiple}`,
+      youAll: `were ${parts.pastParticiple}`,
+      they: `were ${parts.pastParticiple}`,
+    },
+    simple: {
+      I: `was ${parts.pastSimple}`,
+      you: `were ${parts.pastSimple}`,
+      it: `was ${parts.pastSimple}`,
+      we: `were ${parts.pastSimple}`,
+      youAll: `were ${parts.pastSimple}`,
+      they: `were ${parts.pastSimple}`,
+    },
+    perfect: {
+      I: `was ${parts.pastParticiple}`,
+      you: `were ${parts.pastParticiple}`,
+      it: `was ${parts.pastParticiple}`,
+      we: `were ${parts.pastParticiple}`,
+      youAll: `were ${parts.pastParticiple}`,
+      they: `were ${parts.pastParticiple}`,
+    },
+    perfectContinuous: {
+      I: `was ${parts.pastParticiple}`,
+      you: `were ${parts.pastParticiple}`,
+      it: `was ${parts.pastParticiple}`,
+      we: `were ${parts.pastParticiple}`,
+      youAll: `were ${parts.pastParticiple}`,
+      they: `were ${parts.pastParticiple}`,
+    }
```

Comment:
Good catch with the initialization!!

Similar to the above.  Lets let the loop do the repetitious part for us

```suggestion
    continuous: {},
    simple: {},
    perfect: {},
    perfectContinuous: {}
```